Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -25,6 +25,12 @@ public static class PythonExtensions {
 
 namespace Microsoft.Dafny.Compilers {
   public class PythonCompiler : SinglePassCompiler {
+    public override void OnPreCompile(ErrorReporter reporter, ReadOnlyCollection<string> otherFileNames) {
+      base.OnPreCompile(reporter, otherFileNames);
+      if (DafnyOptions.O.CoverageLegendFile != null) {
+        Imports.Add("DafnyProfiling");
```

Comment:
Should this go in `DafnyRuntime`?

Reply:
I think the idea is that you get to design the logging output, by making it an extern.